File: server/eBooks.Database/Models/PublisherFollow.cs
using System;
using System.Collections.Generic;

namespace eBooks.Database.Models;

public partial class PublisherFollow
{
    public int UserId { get; set; }

    public int PublisherId { get; set; }

    public DateTime ModifiedAt { get; set; }

    public virtual User Publisher { get; set; } = null!;

    public virtual User User { get; set; } = null!;
}

File: client/ebooks_user/lib/models/publisher_follow/publisher_follow.dart
import "package:ebooks_user/models/users/user.dart";
import "package:json_annotation/json_annotation.dart";

part "publisher_follow.g.dart";

@JsonSerializable()
class PublisherFollow {
  int? userId;
  User? publisher;
  DateTime? modifiedAt;

  PublisherFollow({this.userId, this.publisher, this.modifiedAt});

  factory PublisherFollow.fromJson(Map<String, dynamic> json) =>
      _$PublisherFollowFromJson(json);
  Map<String, dynamic> toJson() => _$PublisherFollowToJson(this);
}
